### ISSUE TITLE
Add option to remove '- Mozilla Firefox' from title

### ISSUE THREAD

**Issue by Mehradzie** [hrvojesolc/Custom-Titlebar-Text#7]
Somehow, some websites show really strange domain when the custom settings is set to {page domain}
per say the following
http://www.altium.com/documentation/15.1/display/ADES/((Defining+Board+Regions+and+Bending+Lines))_AD

gives the result
`com - Mozilla Firefox`

BTW, is there a way to get rid of the ` - Mozilla Firefox` post-fix to every url in titlebar? I am using 57Beta

**Comment by hr3x8** [comment]
Hello Mehradzie,

Domain request is a duplicate of #4. I've created that one myself and am already aware of the issue. Planning to resolve it within a week or so.

Unfortunately using Web Extensions API there is currently no way to get rid of <code>- Mozilla Firefox</code> as of now. I've renamed the issue to <code>Add option to remove '- Mozilla Firefox' from title</code> to better reflect outstanding request. I will revisit it if/when it becomes an option in Web Extensions API. Currently it is not there are <code>- Mozilla Firefox</code> gets added after the fact.

Thanks,

Hrvoje
